src/app/api/repositories/[owner]/[repo]/preferences/route.ts

```
@@ -0,0 +1,63 @@
+        userId: dbUser.id,
+        repository,
+        isActive: updateData.isActive ?? false,
+        ignoredTypes: updateData.ignoredTypes ?? "Issue,PullRequest,Commit",
+      },
+    });
+
+    return Response.json({
+      ...preference,
+      ignoredTypes: preference.ignoredTypes.split(","),
+    });
+  } catch (error) {
```

ESLint

[error] 'error' is defined but never used.

(@typescript-eslint/no-unused-vars)